.github/workflows/test.yml

```diff
@@ -1,50 +1,55 @@
 name: Run tests
 
 on:
-  workflow_dispatch:
+  workflow_call:
     inputs:
       environment:
-        type: string
-        description: Name of the environment to use
         required: true
-      release_body:
         type: string
-        description: GitHub Release Message
-        default: "GitHub Release"
 
-  push:
-    branches: [ '*' ]
-    environment:
-      name: Nam
+    secrets:
+      C8Y_PASSWORD:
+        required: true
```

Comment:
why is this required as input?

Reply:
Because the tests can not be run without password, I add required as True although it is not necessary

Reply:
but what kind of password is that? where is it used? shouldnt we use the password from the environment?

Reply:
Yes, the Password from environment is get in the build.yml and passed to this test.yml

Reply:
https://docs.github.com/en/actions/deployment/targeting-different-environments/using-environments-for-deployment#environment-secrets

Note: Workflows that run on self-hosted runners are not run in an isolated container, even if they use environments. Environment secrets should be treated with the same level of security as a repository and organization secrets. For more information, see "[Security hardening for GitHub Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#hardening-for-self-hosted-runners)."